src/main/python/README.md

```diff
@@ -0,0 +1,14 @@
+### Requirements
+
+Python>=3.5
+`pip install pyyaml`
+
+### Before Run
+
+Set Anserini root in the yaml file as `default->root`
```

Comment:
One suggestion: since everyone will likely have a different path for Anserini, they'll need to change this config option in the yaml file. Which means that git will mark this tracked file as modified.

What if instead we create a template `all.yaml.example` file, version track it, and get the user to copy the `all.yaml.example` to `all.yaml` and have them change `all.yaml`? We add `all.yaml` to `.gitignore` so that the git stage is clean.

Reply:
A more robust solution may be [YAML Loader](https://gist.github.com/joshbode/569627ced3076931b02f)? We can import the shared and tracked `all.yaml.example` file into our own local and untracked `all.yaml` file, eliminating mistakes from copy and paste.

Reply:
Ah, I see. A global tracked `yaml` file with a smaller untracked user-specific `yaml` file that just does the override for the particular configuration maybe a better idea, yes, if I understood correctly.

Reply:
Let's not over-engineer? Since running regressions on tuna is the common case, let's hard-code the yaml with the tuna path. Let's then pass in an override path on the command line?

Reply:
@lintool The corpus path is indeed hardcode with tuna path. 
Here they are suggesting to use a template to reflect the Anserini path. This is a reasonable suggestion as everyone will have their own Anserini root. Changing the yaml file will introduce a diff.